src/ResourceManager/Compute/Commands.Compute/help/Invoke-AzureRmVmssServiceFabricUDWalk.md

```diff
@@ -0,0 +1,213 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/invoke-azurermvmssservicefabricudwalk
+schema: 2.0.0
+---
+
+# Invoke-AzureRmVmssServiceFabricUDWalk
+
+## SYNOPSIS
+Manual platform update domain walk to update virtual machines in a service fabric virtual machine scale set.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Invoke-AzureRmVmssServiceFabricUDWalk [-ResourceGroupName] <String> [-VMScaleSetName] <String>
+ [-PlatformUpdateDomain] <Int32> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Invoke-AzureRmVmssServiceFabricUDWalk [-PlatformUpdateDomain] <Int32> -ResourceId <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ObjectParameter
+```
+Invoke-AzureRmVmssServiceFabricUDWalk [-PlatformUpdateDomain] <Int32>
+ -VirtualMachineScaleSet <PSVirtualMachineScaleSet> [-AsJob] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Invoke manual platform update domain walk to update virtual machines in a service fabric virtual machine scale set.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Invoke-AzureRmVmssServiceFabricUDWalk -ResourceGroupName $rgname -VMScaleSetName $vmssName -PlatformUpdateDomain 0
+```
+
+This invokes service fabric update walk on UD 0 for the virtual machine scale set specified by resource group name and scale set name.
+
+### Example 2
+```
+PS C:\> Invoke-AzureRmVmssServiceFabricUDWalk -VirtualMachineScaleSet $vmss -PlatformUpdateDomain 1
```

Comment:
@hyonholee for the examples you've included (specifically this one), it would be nice to include lines before the command that are setting up the variables you are using (such as `$vmss`) -- that way, they know what commands they can use to set the values for these variables (or what sample values should be for strings)